src/poisson/abstract_poisson_process.jl

```diff
@@ -24,10 +24,11 @@ end
 
 ground_intensity(pp::AbstractPoissonProcess, t, h) = ground_intensity(pp)
 mark_distribution(pp::AbstractPoissonProcess, t, h) = mark_distribution(pp)
+mark_distribution(pp::AbstractPoissonProcess, t) = mark_distribution(pp) # For simulate_ogata
```

Comment:
Are there other spots where we should add this? For instance wrappers like `BoundedPointProcess`